rdkafka-sys/update-bindings.sh

```diff
@@ -9,13 +9,14 @@ bindgen \
     --no-doc-comments \
     --no-layout-tests \
     --rustified-enum ".*" \
-    --whitelist-function "rd_kafka.*" \
-    --whitelist-type "rd_kafka.*" \
-    --whitelist-var "rd_kafka.*|RD_KAFKA_.*" \
-    --no-recursive-whitelist \
-    --blacklist-function "rd_kafka_conf_set_open_cb" \
-    --raw-line "use libc::{FILE, sockaddr, size_t, ssize_t, c_int, c_void, c_char};" \
+    --allowlist-function "rd_kafka.*" \
+    --allowlist-type "rd_kafka.*" \
+    --allowlist-var "rd_kafka.*|RD_KAFKA_.*" \
+    --no-recursive-allowlist \
+    --blocklist-function "rd_kafka_conf_set_open_cb" \
+    --raw-line "use libc::{FILE, sockaddr, c_int, c_void, c_char};" \
     --raw-line "use num_enum::TryFromPrimitive;" \
+    --default-macro-constant-type "signed" \
```

Comment:
Was this important for this PR? Or just avoiding an `as i32` somewhere?

Reply:
It was avoiding an `as i32` in my code as well as in  existing code. Overall this seems to be how macro constants are generally used in librdkafka, so that seems to be the right way to represent these.